acquia/acquia_cms
# ACMS-1290: Refactored roles & permissions and update PHPUnit tests.

**Motivation**
- Refactor roles & permissions.
- Refactored & updated PHPUnit tests.

**Proposed changes**
Covers changes for following tickets:
- [ACMS-1195](https://backlog.acquia.com/browse/ACMS-1195)
- [ACMS-1316](https://backlog.acquia.com/browse/ACMS-1316)
- [ACMS-1290](https://backlog.acquia.com/browse/ACMS-1290)

**Alternatives considered**
N/A

**Testing steps**
- Run all permission tests and it should pass.
- Verify all roles are rename/refactored as mentioned in [ACMS-1195](https://backlog.acquia.com/browse/ACMS-1195).

**Merge requirements**
